### PR TITLE
feat: translate components part3

### PR DESCRIPTION
**Pull request by flagrede**
Translate organisms components from `SellOrdersTable` to `UserCredits` Translate templates components

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export const LAND_STEWARD_STORY_TITLE_LABEL =
+  'Title of the land stewards story';
+export const LAND_STEWARD_STORY_TITLE_DESCRIPTION =
+  'In one sentence, summarize the story above of the land stewards.';
+export const PROJECT_QUOTE_LABEL =
+  'Quote from land steward or other important stakeholder';
+export const PROJECT_QUOTE_DESCRIPTION =
+  'Choose an inspiring quote that helps others understand why this project is important.';
+export const PROJECT_QUOTE_NAME_LABEL = 'Name of person quoted';
+export const PROJECT_QUOTE_JOB_TITLE_LABEL = 'Title of person quoted';
```
Comment (wgardiner) [inline web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts:15]: Looks like these need to be translated, here with `` msg`...` `` and later when being used in `StoryForm.tsx` with `_(...)` unless theres something I'm missing about how the formik `<Form  translate="true">` component works.
I'm having trouble finding this component in the app, is it still used?

File: web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.constant.ts [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-export const PREFINANCE_TERMS = 'Project prefinance terms';
-export const SEE_PURCHASE_AGREEMENTS = 'see purchase agreement';
-export const SUPPORT_ENABLES = 'What your support enables';
-export const PROJECTED_CREDIT_DELIVERY_DATE = 'projected credit delivery date';
+export const PREFINANCE_TERMS = msg`Project prefinance terms`;
+export const SEE_PURCHASE_AGREEMENTS = msg`see purchase agreement`;
+export const SUPPORT_ENABLES = msg`What your support enables`;
+export const PROJECTED_CREDIT_DELIVERY_DATE = msg`projected credit delivery date`;
 export const PROJECTED_CREDIT_DELIVERY_DATE_TOOLTIP =
   'Delivery dates may vary widely as this project is still in the early stages of development. ';
```
Comment (wgardiner) [inline web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.constant.ts:10]: This needs to be translated

**Review by wgardiner — CHANGES_REQUESTED** [review]
Just a few small changes that need addressing

File: web-marketplace/src/components/organisms/ConfirmationModal.tsx [hunk 3]
```diff
@@ -106,12 +109,16 @@ const ConfirmationModal: React.FC<React.PropsWithChildren<Props>> = ({
           >
             500
           </Title>
-          <Title variant="h6">Regen - Ecocredits</Title>
+          <Title variant="h6">
+            <Trans>Regen - Ecocredits</Trans>
+          </Title>
           <Label size="xs" color="info.main">
-            C01-20190101-20201010-02
+            <Trans>C01-20190101-20201010-02</Trans>
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-marketplace/src/components/organisms/ConfirmationModal.tsx:116]: Correct, I'll fix it.

File: web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts (hunk 1 above)
Comment (flagrede) [reply web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts:15]: It's a legacy form not being used anymore, as per this [comment](https://github.com/regen-network/regen-web/blob/dev/web-marketplace/src/components/organisms/StoryForm.tsx#L157-L159).
Should I delete it or update it anyway @blushi ?

File: web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.constant.ts (hunk 2 above)
Comment (flagrede) [reply web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.constant.ts:10]: Good catch. I'll update it.

**Comment by flagrede** [comment]
@wgardiner The storybook issues will be fixed in another PR (more details in this [comment](https://github.com/regen-network/regen-web/pull/2449#discussion_r1742164866)).

File: web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts (hunk 1 above)
Comment (blushi) [reply web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts:15]: I think we can just delete it

**Review by wgardiner — APPROVED** [review]

**Comment by wgardiner** [comment]
Looks like you just need to resolve conflicts in a few files.

**Pull request merged**